emsdk_manifest.json

```diff
@@ -464,25 +464,25 @@
   {
     "version": "upstream-master",
```

Comment:
Should "upsteam-master" be renamed as well or will that mess things up?

Reply:
I think I'll leave that for now.   This is made of several components some of which do still use master (i.e. emscripten) so it not clear main is better here.